src/pages/Swap/index.tsx

```diff
@@ -359,6 +391,125 @@ export default function Swap() {
       wrapState === WrapState.PENDING) &&
     trade instanceof CoWTrade
 
+  const renderSwapBox = () => (
```

Comment:
we have a PR with extracted Swapbox, it will help clean this up, I think

Reply:
Probably yes, we are waiting for it ;)